docs_source_files/content/support_packages/working_with_cookies.en.md

```diff
@@ -118,7 +118,25 @@ await driver.manage().deleteCookie('cookie-1');
 await driver.manage().deleteAllCookies();
   {{< / code-panel >}}
   {{< code-panel language="kotlin" >}}
-// We don't have a Kotlin code sample yet -  Help us out and raise a PR  
+// Go to the correct domain
+driver.get("http://www.example.com")
+
+// Now set the cookie. This one's valid for the entire domain
+Cookie cookie = Cookie("key", "value")
```

Comment:
Hi @marcus-rp,

This line works in Java but not in kotlin. You please change the code like below

val cookie = Cookie("key", "value")